### PR TITLE
Fix the rootdir in the nouse_install.yml

### PR DESCRIPTION
**Pull request by ThreeMonth03**
Modify ```pytest --rootdir=/tmp -v``` to ```pytest --rootdir=. -v```.

### CODE REVIEW

File: .github/workflows/nouse_install.yml [hunk 1]
```diff
@@ -83,7 +83,7 @@ jobs:
         python3 -c 'import os; print(os.getcwd())'
         python3 -c "import modmesh; print(modmesh.__file__)"
         python3 -c "import _modmesh; print(_modmesh.__file__)"
-        pytest --rootdir=/tmp -v
+        pytest --rootdir=. -v
```
Comment (yungyuc) [inline .github/workflows/nouse_install.yml:90]: Please keep the original code in a comment and add a link to the root cause.  When the bug in pytest is fixed we can change it back.

File: .github/workflows/nouse_install.yml [hunk 2]
```diff
@@ -166,5 +166,5 @@ jobs:
           python3 -c 'import os; print(os.getcwd())'
           python3 -c "import modmesh; print(modmesh.__file__)"
           python3 -c "import _modmesh; print(_modmesh.__file__)"
-          pytest --rootdir=/tmp -v
+          pytest --rootdir=. -v
```
Comment (yungyuc) [inline .github/workflows/nouse_install.yml:177]: Change both places.

**Review by yungyuc — CHANGES_REQUESTED** [review]
- [ ] Make it explicit that `--rootdir=.` is a workaround.

File: .github/workflows/nouse_install.yml [hunk 3]
```diff
@@ -83,7 +83,11 @@ jobs:
         python3 -c 'import os; print(os.getcwd())'
         python3 -c "import modmesh; print(modmesh.__file__)"
         python3 -c "import _modmesh; print(_modmesh.__file__)"
-        pytest --rootdir=/tmp -v
+        ### The following command is the original commend, and it will fail on pytest == 8.0.0 .
+        ### pytest --rootdir=/tmp -v
+        ### Here is the issue and temporary solution: https://github.com/pytest-dev/pytest/issues/11781  
+        ### The alternative command to solve the issue is ```pytest --rootdir=. -v```.
+        pytest --rootdir=. -v
```
Comment (ThreeMonth03) [inline .github/workflows/nouse_install.yml]: I add the comment, including the original command, the modified command, and the source of the issue and solutions.

File: .github/workflows/nouse_install.yml [hunk 4]
```diff
@@ -166,5 +170,9 @@ jobs:
           python3 -c 'import os; print(os.getcwd())'
           python3 -c "import modmesh; print(modmesh.__file__)"
           python3 -c "import _modmesh; print(_modmesh.__file__)"
-          pytest --rootdir=/tmp -v
+          ### The following command is the original commend, and it will fail on pytest == 8.0.0 .
+          ### pytest --rootdir=/tmp -v
+          ### Here is the issue and temporary solution: https://github.com/pytest-dev/pytest/issues/11781  
+          ### The alternative command to solve the issue is ```pytest --rootdir=. -v``` .
+          pytest --rootdir=. -v
```
Comment (ThreeMonth03) [inline .github/workflows/nouse_install.yml]: I also add the comment here.

**Review by ThreeMonth03 — COMMENTED** [review]
Here are the modifications.

File: .github/workflows/nouse_install.yml (hunk 1 above)
Comment (ThreeMonth03) [reply .github/workflows/nouse_install.yml:90]: Thank you  for your suggestions. 
I write the comments, including the issue and the original code.

File: .github/workflows/nouse_install.yml (hunk 3 above)
Comment (yungyuc) [reply .github/workflows/nouse_install.yml]: Thanks, the messages are clear.  Could you please follow the comment style to use a single `#` symbol?

File: .github/workflows/nouse_install.yml [hunk 5]
```diff
@@ -83,7 +83,11 @@ jobs:
         python3 -c 'import os; print(os.getcwd())'
         python3 -c "import modmesh; print(modmesh.__file__)"
         python3 -c "import _modmesh; print(_modmesh.__file__)"
-        pytest --rootdir=/tmp -v
+        # The following command is the original commend, and it will fail on pytest == 8.0.0 .
+        # pytest --rootdir=/tmp -v
+        # Here is the issue and temporary solution: https://github.com/pytest-dev/pytest/issues/11781  
+        # The alternative command to solve the issue is ```pytest --rootdir=. -v```.
```
Comment (ThreeMonth03) [inline .github/workflows/nouse_install.yml:89]: I modify the number of ```#``` to keep the consistent coding style.

File: .github/workflows/nouse_install.yml [hunk 6]
```diff
@@ -166,5 +170,9 @@ jobs:
           python3 -c 'import os; print(os.getcwd())'
           python3 -c "import modmesh; print(modmesh.__file__)"
           python3 -c "import _modmesh; print(_modmesh.__file__)"
-          pytest --rootdir=/tmp -v
+          # The following command is the original commend, and it will fail on pytest == 8.0.0 .
+          # pytest --rootdir=/tmp -v
+          # Here is the issue and temporary solution: https://github.com/pytest-dev/pytest/issues/11781  
+          # The alternative command to solve the issue is ```pytest --rootdir=. -v``` .
```
Comment (ThreeMonth03) [inline .github/workflows/nouse_install.yml:176]: I modify the number of ```#``` to keep the consistent coding style.

**Review by ThreeMonth03 — COMMENTED** [review]
I modify the number of ```#``` to keep the consistent coding style.

File: .github/workflows/nouse_install.yml (hunk 3 above)
Comment (ThreeMonth03) [reply .github/workflows/nouse_install.yml]: > Thanks, the messages are clear. Could you please follow the comment style to use a single `#` symbol?

@yungyuc No problem, I modify the inconsistent part and push the modification to the branch.

**Comment by yungyuc** [comment]
Thanks @ThreeMonth03 for working around the pytest issue (https://github.com/pytest-dev/pytest/issues/11781)

**Pull request merged**